src/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -1374,6 +1374,42 @@ public void ICollectionSyncRoot_NotSupported()
             ICollection c = ImmutableArray.Create(1, 2, 3);
             Assert.Throws<NotSupportedException>(() => c.SyncRoot);
         }
+        
+        //[Fact] Perf test
```

Comment:
StyleCop: leave a single space after the `//` or use `////`. 
Also it would be helpful if you put a `//` in between the attribute and the `Perf test` so that folks can uncomment the attribute without introducing a compile error. So in short, can you change each of these lines to:
```csharp
    ////[Fact] // Perf test
```